src/lib/base.jl

```diff
@@ -121,6 +121,11 @@ end
   
   pairs_namedtuple_pullback(dx::NamedTuple) = (dx.data,)
 
+  function pairs_namedtuple_pullback(dx::Tuple) 
+    t0 = isempty(dx) ? () : NamedTuple{N}(values(dx))
+    return (t0,)
+  end
+
```

Comment:
```suggestion
pairs_namedtuple_pullback(dx::Tuple{}) = (NamedTuple(),)
```
If this works, better do the most minimal change that solves your issue

Reply:
That solve it.